src/components/ProjectGroup/ProjectGroup.js

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import './ProjectGroup.css';
+import ProjectCard from '../ProjectCard/ProjectCard'
+
+function ProjectGroup({name, projects}) {
+  return (
+    <div className="ProjectGroup">
+      <h2 className="ProjectGroup-Name">{name}</h2>
+      <div className="ProjectCards">
+        {projects ? projects.map((project, idx) => <ProjectCard key={idx} title={project.title} desc={project.description} url={project.url} />) : ''}
```

Comment:
__*Preference*__
in the case of brevity since you are mostly just passing the project down as props you can do `<ProjectCard key={project.title" {...project} />` that will pass each of the project's properties down as a prop. (This would require you to change the prop on the project card from desc to description.